app/modules/settings/plugins/index.js

```diff
@@ -121,6 +124,17 @@ export default angular.module('settings.plugins', [
         title: 'Plugin configuration',
         message: 'Successfully saved'
       }))
+      .error((err) => {
+        console.log('LISTE DES ERREURS = ')
+        forEach(err.data, (data) => {
+          const field = slice(data.field.split('.'), 2)
+          console.log('field = ', field)
+          let fieldPath = data.field.split('.')[1]
+          forEach(field, (name) => fieldPath += ' > ' + name)
```

Comment:
Also, do not display directly the name of the field, some fields have an associated title which would be better appropriate to display.